pom.xml

```diff
@@ -44,18 +54,54 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.30</version>
```

Comment:
you should not specify the version - it is taken from `spring-boot-starter-parent`